src/take.ts

```diff
@@ -54,22 +58,16 @@ export async function handleTakes({
         config,
       });
     }
-    // Delay between handling each liquidation, probably superfluous
-    await delay(config.delayBetweenActions);
   }
```

Comment:
Can we keep this delay, let say there are multiple takeable loans in auction, can we insure the bot will have a pause before interactin with 1 inch...?